schemas/IDLs/org.gel.models.report.avro/4.1.0/ExitQuestionnaire.avdl

```diff
@@ -0,0 +1,248 @@
+        * `not_yet_actioned`: not yet actioned, but potentially actionable in the future
+    */
+    enum CancerUsability {already_actioned, actioned_result_of_this_WGA, not_yet_actioned}
+
+    /**
+    An enumeration Variant tested:
+        * `not_indicated_for_patient_care`: No: not indicated for patient care at this time
+        * `no_orthologous_test_available`: No: no orthologous test available
+        * `test_performed_prior_to_WGA`: Yes: test performed prior to receiving WGA (eg using standard-of-care assay such as panel testing, or sanger sequencing)
+        * `technical_validation_following_WGA`: Yes: technical validation performed/planned following receiving this WGA
+    */
+    enum CancerTested {not_indicated_for_patient_care, no_orthologous_test_available, test_performed_prior_to_WGA , technical_validation_following_WGA}
+
+    record CancerCaseLevelQuestions{
+
+    /**
+    Total time taken to review/collate evidence for variants (hours).
+    Include all literature review time, consultation with relevant experts etc.
+    */
+    double total_time;
+
+    /**
+    Time taken to discuss case at MDT (hours).
+    */
+    double mdt1_time;
+
+    /**
+    If the case is discussed at a 2nd MDT please enter time here (hours).
+    */
+    union {null, double} mdt2_time;
+
+    /**
+    Total time to design ALL validation assay(s) for case (hours).
+    Only applicable if it is necessary to design a new assay to validate the variant.
+    */
+    union {null, double} validation_assay_time;
+
+    /**
+    Technical Laboratory Validation. Total time for validation test wet work for all variants (hours).
+    */
+    union {null, double} wet_validation_time;
+
+    /**
+    Analytical Laboratory Validation. Total time for analysis of validation results for all variants (hours).
+    */
+    union {null, double} analytical_validation_time;
+
+    /**
+    Primary Reporting. Time taken to complete primary reporting stage (hours).
+    */
+    double primary_reporting_time;
+
+    /**
+    Report Authorisation. Time taken to check and authorise report (hours).
+    */
+    double primary_authorisation_time;
+
+    /**
+    Report Distribution.
+    Please enter, where possible/accessible how long it takes for the result to be conveyed to the patient.
+    E.g. via letter from the clinician (days).
+    */
+    double report_distribution_time;
+
+    /**
+    Total time from result to report.
+    The total time taken from when the analysis of the WGS results started  to a report being received
+    by the patient include any 'waiting' time (days).
+    */
+    double report_distribution_time;
+
+    /**
+    Which parts of the WGA were reviewed?
+    */
+    CancerReviewedMDTWGA reviewedInMdtWga;
+
+    /**
+    Were potentially actionable variants detected?
+    */
+    CancerActionableVariants actionableVariants;
+
+    }
+
+
+    record CancerVariantLevelQuestions{
```

Comment:
Do we have any field for identifying the variant in cancer cases? I.e. if the user adds their comments for some variant, it looks like there is no way to identify it in the answer that we send to the server